build.gradle

```diff
@@ -34,20 +34,20 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-webflux'
 	compileOnly 'org.projectlombok:lombok'
 	annotationProcessor 'org.projectlombok:lombok'
-	implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testImplementation 'de.flapdoodle.embed:de.flapdoodle.embed.mongo'
 	testImplementation 'io.projectreactor:reactor-test'
+	implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
 
-	testImplementation('com.codeborne:selenide:5.25.0')
-	testImplementation("io.qameta.allure:allure-selenide:${allureVersion}")
-	testImplementation('org.assertj:assertj-core:3.21.0')
-	testImplementation('com.google.truth:truth:1.1.3')
-	testImplementation('org.hamcrest:hamcrest:2.2')
-	testImplementation('org.slf4j:slf4j-simple:1.7.30')
-
-	testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
-	testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${junitVersion}")
+//	testImplementation('com.codeborne:selenide:5.25.0')
```

Comment:
Need to review frontend/end-to-end testing methods. Selenide seems to be causing issues.